docs/using/tutorial/moving-volume-tutorial.rst

```diff
@@ -0,0 +1,193 @@
+==============================
+Tutorial: Moving a Data Volume
+==============================
+
+.. note:: This tutorial takes roughly 30 minutes, but because there are a few things to download, times might vary depending on the speed of your connection.
+
+.. contents:: 
+   :local:
+   :backlinks: none
+   :depth: 2
+
+You will use Flocker to migrate a Docker container with its data volume from one node to another.
+The container you move will be part of a two-container application, the other container will not move and the two will remain connected even when they are on different hosts.
+
+To begin the tutorial you will first install the Flocker client on your local machine, then install Flocker onto two hosts.
+You will then be ready to use Flocker to migrate a Docker container with a volume attached from one host to the other.
+
+You will be controlling your Flocker cluster via the CLI that you will have installed locally.
+The following diagram illustrates the initial server-side Flocker setup that you will control via the CLI:
+
+.. image:: images/flocker-tutorial-initial-setup.svg
+   :alt: In the initial server-side Flocker setup there are two servers, one of which has two Docker containers running; one container is a running a web application, the other has a Redis database with a volume.
+   :align: center
+
+The following diagram illustrates how the server-side Flocker setup will be configured at the end of the tutorial:
+
+.. image:: images/flocker-tutorial-final-setup.svg
+   :alt: Following the completion of this tutorial the server-side Flocker setup will be configured with the web application still running within a container on the first server, while the Redis server with a volume is running on the second server.
+   :align: center
+
+Flocker manages the data migration and the link between the two containers.
+
+If you have any feedback or problems, you can :ref:`talk-to-us`.
+
+Before You Begin
+================
+
+Requirements
+------------
+
+To replicate the steps demonstrated in this tutorial, you will need:
+
+* Ubuntu 14.04, Ubuntu 15.04 or OS X with `Homebrew`_ installed.
+* Two Virtual Machines (VMs) on your local machine.
+  For this tutorial, you are supplied with Vagrant images to create the tutorial environment on VMs using VirtualBox, so you must have `Vagrant`_ and `VirtualBox`_ installed.
+* At least 4GB RAM available.
+
+Installing the Client
+=====================
+
+The Flocker client runs locally on your machine, and will control the two instances of Flocker located on the hosts.
+To install the Flocker client, run the following in your terminal:
+
+OS X
+----
+
+.. task:: test_homebrew flocker-|latest-installable|
+   :prompt: you@laptop:~$
+
+Ubuntu 14.04 and 15.04
+----------------------
+
+.. task:: install_cli ubuntu-14.04
+   :prompt: you@laptop:~$
+
+.. The 14.04 instructions are currently the same as the 15.04 instructions, so the task link below is commented out.
```

Comment:
How will we know? Does a developer / reviewer who may have never seen this comment have to remember? I think keep a separate tab and then make an issue to combine them - that issue can come with a test that they are all the same.